packages/fast-components/src/slider/slider.template.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from "@microsoft/fast-element";
+import { Slider } from "./slider";
+import { ref } from "@microsoft/fast-element/dist/directives";
+
+export const SliderTemplate = html<Slider>`
+  <template
+    role="slider"
+    aria-valuenow="${x => x.value}"
+    aria-valuemin="${x => x.min}"
+    aria-valuemax="${x => x.max}"
+  >
+    <div
+          ${ref("root")}
+          part="slider"
+          class="slider"
+    >
+      <div part="layout-region" class="layout-region">
```

Comment:
we use `positioning region` in dialog, should we align?

Reply:
Sounds good, we shall align.

Reply:
done